src/api/request-hooks/hook.ts

```diff
@@ -0,0 +1,59 @@
+import {
+    RequestFilterRule,
+    ConfigureResponseEvent,
+    ConfigureResponseEventOptions,
+    RequestEvent,
+    ResponseEvent
+} from 'testcafe-hammerhead';
+import { castArray } from 'lodash';
```

Comment:
Could you please separate multiline imports with an empty line?

```suggestion

import { castArray } from 'lodash';
```